features/admin.py

```diff
@@ -50,6 +51,11 @@ def get_extra(self, request, obj=None, **kwargs):
         return self.extra - obj.opening_hours.count() if obj else self.extra
 
 
+class PriceListInLine(TranslatableTabularInline):
+    model = PriceTag
+    extra = 0
+
+
```

Comment:
Nitpicking, but should these be in alphabetical order?

Reply:
fixed.